_security/configuration/best-practices.md

```diff
@@ -0,0 +1,125 @@
+
+Whenever possible, adhere to these recommendations:
+
+- Set `audit.log_request_body` to `false`.
+- Set `audit.resolve_bulk_requests` to `false`.
+- Enable `compliance.write_log_diffs`.
+- Minimize entries for `compliance.read_watched_fields`.
+- Minimize entries for `compliance.write_watched_indices`.
+
+## 7. Consider disabling the private tenant
+
+In many cases, the use of private tenants is unnecessary, yet this feature is enabled by default. As a result, every OpenSearch Dashboards user is provided with their own private tenant and a corresponding new index in which to save objects. This can lead to a large number of unnecessary indexes. Evaluate whether this feature is needed, and, if it isn't, disable it by adding the following configuration to the `config.yml` file:
+
+```yaml
+config:
+  dynamic:
+    kibana:
+      multitenancy_enabled: true
+      private_tenant_enabled: false
+```
+
+## 8. Manage the configuration using securityadmin.sh
+
+Use `securityadmin.sh` to manage the configuration of your clusters. `securityadmin.sh` is a command-line tool provided by OpenSearch for managing security configurations. It allows administrators to efficiently manage security settings, including roles, role mappings, and other security-related configurations within an OpenSearch cluster.
+
+Before making changes with `securityadmin.sh`, always create a backup of the current configuration to prevent potential loss. For more detailed information, visit the [security admin configuration](https://opensearch.org/docs/latest/security/configuration/security-admin/) section.
+
+### Benefits of using securityadmin.sh
+
+1. Consistency: By using `securityadmin.sh`, administrators can ensure consistency across security configurations within a cluster. This helps to maintain a standardized and secure environment.
+2. Automation: `securityadmin.sh` enables automation of security configuration tasks, making it easier to deploy and manage security settings across multiple nodes or clusters.
+3. Version control: Security configurations managed through `securityadmin.sh` can be version-controlled using standard version control systems like Git. This facilitates tracking changes, auditing, and reverting to previous configurations.
+
+To prevent configuration overrides, it's important to note that changes made through the OpenSearch Dashboards UI or using the OpenSearch API provide a convenient way to modify security settings. To ensure that these updates are preserved when using the `securityadmin.sh` tool, first create a backup of the current configuration. This ensures that all configurations are captured before uploading the modified configuration with `securityadmin.sh`.
```

Comment:
I don't believe my comment here was addressed.

Reply:
Suggestion: Maybe we can recast the first sentence as something like "You can prevent configuration overrides by..."

Reply:
Complete and reworded.

You can prevent configuration overrides by ensuring any updates made through OpenSearch Dashboards UI or via the OpenSearch API are maintained through the use of the `securityadmin.sh` tool. A backup of configuration should first be made to ensure all configuration are captured prior to uploading the modified configuration using `securityadmin.sh`.